### PR TITLE
Created Multiple Images Files For Products!

### PR DESCRIPTION
**Pull request by Dev-PhyoeWai**

### CODE REVIEW

File: app/Http/Controllers/ProductController.php [hunk 1]
```diff
@@ -2,68 +2,108 @@
...
+    {
+        $request->validate([
+            'name' => 'required|string|max:255',
+            'type' => 'required|string|max:255',
+            'price' => 'required|numeric',
+            'quantity' => 'required|integer',
+            'images.*' => 'image|mimes:jpeg,png,jpg,gif,svg|max:2048'
+        ]);
+
+        $product = Product::create($request->only('name', 'type', 'price', 'quantity'));
```
Comment (Mawite01) [inline app/Http/Controllers/ProductController.php:31]:
```suggestion
        $product = Product::create($request->all());
```

Comment (Mawite01) [inline app/Http/Controllers/ProductController.php:31]:
```suggestion
        $product = Product::create($request->validated());
```

Comment (Dev-PhyoeWai) [reply app/Http/Controllers/ProductController.php:31]: Updated!!

Comment (Dev-PhyoeWai) [reply app/Http/Controllers/ProductController.php:31]: Updated!!

Comment (Dev-PhyoeWai) [reply app/Http/Controllers/ProductController.php:31]: Updated!!

**Pull request merged**